app/src/main/java/org/dslul/openboard/inputmethod/keyboard/internal/KeyboardTextsTable.java

```diff
@@ -799,9 +799,9 @@ public static String[] getTextsTable(final Locale locale) {
         null, null, null, null,
         /* ~ morekeys_u */
         // Label for "switch to alphabetic" key.
-        // U+0995: "क" BENGALI LETTER KA
```

Comment:
I had overlooked this: the header of this file says "_!!!!! DO NOT EDIT THIS FILE !!!!!_".

When running `gradle :tools:make-keyboard-text:makeText`, the changes are undone. Better do the changes in `tools/make-keyboard-text/src/main/resources/values-<locale>/donottranslate-more-keys.xml` as indicated on top of the file.

Reply:
my bad. I guess it won't be a problem, as it is just a comment

Reply:
You're right, actually it doesn't matter anyway

Reply:
I couldn't find the source for the wrong letters though. I suspect it's somewhere in `MoreKeysResources`